project_event/models/mail_tracking_value.py

```diff
@@ -46,6 +46,13 @@ def create_tracking_values(self, initial_value, new_value, col_name, col_info):
                 'old_value_char': initial_value and initial_value.name_get()[0][1] or '',
                 'new_value_char': new_value and new_value.name_get()[0][1] or ''
             })
+        elif col_info['type'] == 'many2many':
+            values.update({
+                'old_value_char': initial_value and list(
+                    map(lambda x: x.name, initial_value)) or '',
+                'new_value_char': new_value and list(
```

Comment:
', '.join(list(map(lambda employee: employee.name, new_value))) need to use this one; otherwise, you are passing a list with ''['  Employees   "]"". We dont want the square brackets